webapp/src/pages/web/auth/login.tsx

```diff
@@ -1,24 +1,29 @@
 import Head from 'next/head';
 import { LoginForm } from 'dogma/features/auth/LoginForm';
-import { useAppSelector } from 'dogma/store';
+import { useAppDispatch, useAppSelector } from 'dogma/store';
 import { useEffect } from 'react';
 import { useRouter } from 'next/router';
+import { login } from 'dogma/features/auth/authSlice';
 
 const LoginPage = () => {
   const router = useRouter();
-  const user = useAppSelector((state) => state.auth.user);
+  const sessionId = useAppSelector((state) => state.auth.sessionId);
   useEffect(() => {
-    if (user) {
+    if (sessionId) {
       router.push('/');
     }
-  }, [router, user]);
+  }, [router, sessionId]);
```

Comment:
If `isInAnonymousMode` is set to `true`, should we redirect to `/`?

Reply:
Good catch, thank you!